docs/install/using-visual-studio-vm.md

```diff
@@ -95,7 +95,9 @@ The spectrum of development environments is huge, and there’s real cost associ
 
 A quick summary: Use the System Preparation tool (Sysprep) and shut down the running VM, and then capture *(Figure 1)* the VM as an image through the UI in the Azure portal. Azure saves the `.vhd` file that contains the image in the storage account of your choosing. The new image then shows up as an Image resource in your subscription’s list of resources.
 
-<img src="media/capture-vm.png" alt="Capture an image through the Azure portal’s UI" style="border:3px solid Silver; display: block; margin: auto;"><center>*(Figure 1) Capture an image through the Azure portal’s UI.*</center>
+![Capture an image through the Azure portal’s UI](media/capture-vm.png)
```

Comment:
This loses the border, but doesn't look like this style is used anywhere else.

Reply:
@TerryGLee Are you okay with losing the border?

Reply:
Although we're supposed to include a grey border around any screenshot that includes white space, this is not the way to do it, so the change is fine. (The content on this page is a contribution by an engineering PM; he's out of the office all this week so let's go ahead and make the change and I'll let him know later that we made it and why.)